Makefile

```diff
@@ -1,19 +1,12 @@
-dev: check-env
+dev:
 	cd .. && pip3 uninstall -y sdx-common && pip3 install -I ./sdx-common
 	pip3 install -r requirements.txt
 
 build:
-	pip3 install -I -r requirements.txt
-	git clone https://github.com/ONSdigital/sdx-common.git
```

Comment:
We still need to be getting sdx-common from somewhere. The agreed way in a dev environment is to git clone into the root of the directory then point pip at it, i.e:

```
$ git clone https://github.com/ONSdigital/sdx-common.git
$ pip install git+file://sdx-common.git@0.7.0
```